# Occupancy Quadtree/Octree Message
std_msgs/Header header
uint8           dim
string          tree_type
bool            is_double
bool            binary
int8[]          data
